0001-use-architectural-decision-records.md

```diff
@@ -0,0 +1,27 @@
+# Use Architectural Decision Records
+
+## Context
+
+We need a way to record technical decisions not only in projects but for the team more widely. To provide some consistency we suggest using [Architectual Decisions Records](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions) to record our decisions as a team.
+
+Architectural decision records are essentially a formatted log of technical decisions. We provide [a template](TEMPLATE.md) that the team can use for recording these decisions.
+
+## Decision
+
+We will use architectural decision records (ADRs) to record our technical decisions on projects and as a team. When we make a new technical decision, we will create a log using the TEMPLATE.md file and add it to the README.md in the [technical-decisions](https://github.com/WeAreSnook/tech-decisions) repo.
+
+The first technical decision [Github vs Gitlab](0000-github-vs-gitlab.md) will keep its initial formatting.
```

Comment:
I think a ticket should be written to rewrite github-vs-gitlab using this new format. 

It being the first decision it will receive more views than some of the others, and it is important it sets the scene for other decisions.

I'm happy to make these changes, if you could create the ticket.

Reply:
Hi Toby, I've created a ticket for this if you could pick that up. Anything that's not clear let me know. Welcome your feedback on any tickets :)